docs/docs/creating-and-modifying-pages.md

```diff
@@ -92,13 +92,103 @@ exports.createPages = async ({ graphql, actions, reporter }) => {
       // In your blog post template's graphql query, you can use pagePath
       // as a GraphQL variable to query for data from the markdown file.
       context: {
-        pagePath: path,
-      },
+        pagePath: path
+      }
     })
   })
 }
 ```
 
+## Trade-offs of querying for all fields in the context object of `gatsby-node.js`
+
+Imagine a scenario where you could query for all the parameters your template would need in the `gatsby-node.js`. What are the performance advantages this method might propose? In this section, you will look into this.
```

Comment:
```suggestion
Imagine a scenario where you could query for all the parameters your template would need in the `gatsby-node.js`. What would the implications be? In this section, you will look into this.
```

Performance advantages is 1 of 2 things addressed in the sections below (performance and incremental builds) so I think rewording this would make sense.

Reply:
Oh, I agree.